content/en/blog/_posts/2021-01-28-PSP-deprecation.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: blog
+title: 'What’s Up with Pod Security Policies?'
+date: 2021-02-04
+slug: psp-update-2021
+---
+
+**Authors:** Contributor Comms Team
+
+_The following is a brief overview of upcoming changes to Kubernetes. Bookmark and [keep an eye on our release site](https://www.kubernetes.dev/resources/release/) to see the latest news and other important updates._
+Pod Security Policies (PSP), starting with the Kubernetes 1.21, will begin the [process of deprecation](/docs/reference/using-api/deprecation-policy/) with the intention to fully remove it in a future release. This prompts the question: what will replace Pod Security Policies in the future? In short, [Admission Controllers](/docs/reference/access-authn-authz/admission-controllers/). For more details, keep reading.
+
+
+## What do Pod Security Policies do now?
+
+> A Pod Security Policy is a cluster-level resource that controls security sensitive aspects of the pod specification. The [PodSecurityPolicy](/docs/reference/kubernetes-api/policies-resources/pod-security-policy-v1beta1/) object defines a set of conditions that a pod must run with in order to be accepted into the system, as well as defaults for the related fields. They allow an administrator to control a number of security-related functions.
+
+The [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)
+documentation explains this in more detail.
+
+
+## Why are Pod Security Policies being deprecated? 
+
+To watch an explanation of PSP removal, you can check out this session by SIG-Auth at KubeCon NA 2019:
```

Comment:
nit: "To watch an explanation of PSP removal ..."? Possibly reword to  something like:
To [better] understand the development of PSP ... and why PSP is being deprecated OR
To understand why PSP ...

Since the video is from 2019, I am guessing that your intent is to provide some historical perspective about the upcoming changes. This section describes the basic rationale for removing or deprecating PSP (without going into much technical detail). The video is backing up your claim and reasoning for the taking these steps to change.